app.rb

```diff
@@ -80,6 +80,19 @@ class MakersBnb < Sinatra::Base
   get '/rooms/new' do
     erb :new
   end
+  
+  get '/rooms/manage' do
+    @user = User.find_id(session[:id])
+    @rooms = Room.all
+    erb :manage_rooms
+  end
+  
+  post '/rooms/delete' do
+  p params
```

Comment:
We should remove debug statements before we merge into main

Reply:
can you delete?